tests/testthat/assets/sidebar/README.md

```diff
@@ -0,0 +1,93 @@
+
```

Comment:
Can you please massively simplify this file so it only contains the components that are important?